### DISCUSSION TITLE
Increase the storage capacity

### DISCUSSION THREAD

**Discussion by PrabhuShanmughapriyan** [postalserver/postal#2674] [Help with using Postal] [answered]
Hi,

I am using postal for past few weeks.. So far i love it.. I was able to externalize the database and RabbitMQ so it - postal-worker containers can be run on different servers - hence achieving HA and horizontal scaling. 

I have few questions to get answered before moving to production
- I am planning to use my cloud provided MariaDB. that being said, can i have indefinite storage - instead of 30 days.
- Can i integrate Postal with DoveCot (IMAP) so it behaves as a complete email server solution

**Comment by willpower232** [comment, marked as answer]
Postal isn't designed for indefinite storage so you could set the number really high but fundamentally, the software you're using to put messages into Postal is the software that should be recording your history of sending emails. It will be better at managing the extra context around the message than Postal ever would.

You may need to implement webhooks to add more information about how the email did on its sending to your software.

Postal can relay via other services or you could set up credentials and routes to send via Postal from Dovecot but it was mostly intended for transactional purposes so you might struggle to deal with the back and forth.

If you need an inbox, you might not want to use Postal at all.

**Reply by PrabhuShanmughapriyan** [reply]
Thank you @willpower232
